src/Sulu/Bundle/TagBundle/Domain/Event/TagRemovedEvent.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TagBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+
+class TagRemovedEvent extends DomainEvent
+{
+    /**
+     * @var int
+     */
+    private $tagId;
+
+    /**
+     * @var int
+     */
+    private $tagName;
+
+    /**
+     * @var bool
+     */
+    private $tagWasMerged;
+
+    public function __construct(
+        int $tagId,
+        string $tagName,
+        bool $tagWasMerged = false
+    ) {
+        parent::__construct();
+
+        $this->tagId = $tagId;
+        $this->tagName = $tagName;
+        $this->tagWasMerged = $tagWasMerged;
+    }
+
+    public function getEventType(): string
+    {
+        return 'removed';
+    }
+
+    public function getEventContext(): array
+    {
+        return [
+            'tagWasMerged' => $this->tagWasMerged,
```

Comment:
should we go here with `merged` or `wasMerged` and are this context `snake_case` or `camelCase`? I think the `tag` prefix is not necessary here or what do you think do we need it?

Reply:
will use `wasMerged`

Reply:
dont have a strong opinion about `snake_case` vs `camelCase`, but we should be consistent once we settled for something 🙂  what do you prefer?

Reply:
In most cases we are using camelCase like `webspaceKey` but know that we did go in the `view` configuration with `snake_case` keys not sure why this was used there. The `wasMerged` is okay for me!